pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3230,6 +3230,7 @@ private void fenceTopicToCloseOrDelete() {
     }
 
     private void unfenceTopicToResume() {
+        subscriptions.values().forEach(sub -> sub.resumeAfterFence());
```

Comment:
resumeAfterFence returns a CompletableFuture, don't we need to "wait"

Reply:
Then you can make resumeAfterFence return "void", can't you ?

Reply:
Good suggestion. Already fixed.